staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2275,6 +2302,9 @@ type Container struct {
 	// More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
 	// +optional
 	Resources ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
+	// Resources resize policy for the container.
+	// +optional
+	ResizePolicy []ResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
```

Comment:
Can you add `+featureGate=<gate>` to all these new fields (just the ones in existing types) please

Reply:
done.

Reply:
fixed in commit https://github.com/kubernetes/kubernetes/pull/102884/commits/b60c5d045bb85eacae064e209994c4afe22a4e5f